tests/installation/await_install.pm

```diff
@@ -81,44 +81,47 @@ sub wait_countdown_stop {
     return wait_screen_change(undef, $stilltime, similarity => $similarity);
 }
 
+sub _set_timeout {
+    my ($timeout) = $_[0];
+    # upgrades are slower
+    ${$timeout} = 5500 if (get_var('UPGRADE') || get_var('LIVE_UPGRADE'));
```

Comment:
I guess that should just be `$timeout` and then return the timeout value at the end of the function

Reply:
the timeout is passed as ref. `$timeout` is the ref and `${$timeout}` points to the value. No need for return as we change the actual value of the ref.